app/models/competition.rb

```diff
@@ -221,6 +221,13 @@ class Competition < ApplicationRecord
   validates :event_restrictions_reason, presence: true, if: :event_restrictions?
   validates_inclusion_of :main_event_id, in: ->(comp) { [nil].concat(comp.persisted_events_id) }
 
+  # Ensure that no string fields exceed the varchar limit in the database
+  columns_hash.each do |column_name, column_info|
+    if column_info.type == :string && column_info.limit
```

Comment:
Probably wanna limit this to the "interesting" columns, i.e. check that `column_name` is included in some pre-defined list.

Reply:
I thought about this as well, but my concern is that future devs may come along and add "interesting" columns, and we have no way of drawing their attention to whether the column should be included in this check (eg, our failing test if you don't add a column to the db sanitizer). 

I agree it's overkill to check _all_ columns, but I don't think there are any explicitly _bad_ things about doing it this way?

Reply:
If a future dev comes along and adds interesting columns, it should also be their responsibility to think about how those interesting columns are being validated.

Actually, I would use your point to form a counter-argument: When introducing a new column, there shouldn't be a (reasonable but) arbitrary validation enforced upon that column out of nowhere

Reply:
Please see latest push - realized we can just add a test for this lol. Thus, the discussion is transmuted into: Which strings should be validated?